cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -214,7 +215,58 @@ fn lower_insn_to_regs<'a>(ctx: Ctx<'a>, inst: IRInst) {
                 unimplemented!("unimplemented lowering for opcode {:?}", op);
             }
         }
-
+        Opcode::Fcopysign => {
+            let dst = output_to_reg(ctx, inst, 0);
+            let lhs = input_to_reg(ctx, inst, 0);
+            let rhs = input_to_reg(ctx, inst, 1);
+            let is64 = flt_ty_is_64(ty.unwrap());
+            if !is64 {
+                let w64 = 0x8000_0000;
```

Comment:
nit: since this is used only once, can you inline it at its use, instead, please?

Reply:
Ok 👍 . Did you think we should fold in rhs as well? None of the other implementations such as Opcode::Iconst that don't fold things in so there would be some inconsistency with the other matches?

Reply:
I think it's reasonable to keep the lhs/rhs variables, since they give slightly more context to what they represent, while `w64`'s value was less clear, in my opinion. I am not sure what you mean by "264", though, feel free to do it too if it makes sense to you!

Reply:
Meant w64 not 264.